src/app/cic/controllers/brpDetails.js

```diff
@@ -4,39 +4,5 @@ const DateControllerMixin = require("hmpo-components").mixins.Date;
 
 const DateController = DateControllerMixin(BaseController);
 
-class BrpDetailsController extends DateController {
-
-  async saveValues(req, res, next) {
```

Comment:
No need for custom `saveValues` if the routing is calculated correctly.

Also, this function isn't calling the [super.saveValues](https://github.com/HMPO/hmpo-form-wizard/blob/master/lib/controller/index.js#L350), so wasn't saving the values correctly, which is where the extra `sessionModel.set` were coming from.